src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1343,6 +1344,7 @@ internal enum ErrorCode
         #region diagnostics for local functions introduced in C# 7
         ERR_DynamicLocalFunctionParamsParameter = 8108,
         ERR_ExpressionTreeContainsLocalFunction = 8110,
+        ERR_LocalFunctionMissingBody = 8112,
```

Comment:
8111 is already reserved in the master brach (it was originally 8111) but it resulted in a merge conflict